src/Workspaces/Core/Portable/Shared/Utilities/IntegerUtilities.cs

```diff
@@ -44,14 +35,7 @@ public static bool HasOneBitSet(long v)
 
         public static int LogBase2(long v)
         {
-            var log = 0;
-
-            while ((v >>= 1) != 0)
-            {
-                log++;
-            }
-
-            return log;
+            return (int)PopCount.LogBase2(v);
```

Comment:
Need to expose LogBase2 in BitOps - this is the second callsite that has used an inferior (perf) approach